### PR TITLE
Support loader

### PR DESCRIPTION
**Pull request by sbosley**
Version of SquidCursorLoader that uses the support library. Refs #44

### CODE REVIEW

File: squidb-addons/squidb-support-loader/src/com/yahoo/squidb/support/SquidCursorLoader.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import com.yahoo.squidb.data.AbstractModel;
+import com.yahoo.squidb.data.SquidCursor;
+import com.yahoo.squidb.data.SquidDatabase;
+import com.yahoo.squidb.sql.Query;
+
+/**
+ * A {@link CursorLoader} that queries a {@link SquidDatabase}. This version of SquidCursorLoader uses the support
+ * library instead of the base Android SDK.
+ */
+public class SquidCursorLoader<T extends AbstractModel> extends AsyncTaskLoader<SquidCursor<T>> {
```
Comment (sbosley) [inline squidb-addons/squidb-support-loader/src/com/yahoo/squidb/support/SquidCursorLoader.java]: @jdkoren do you think this should have the same name as the one in the utility package or should we give it it's own name, like SquidSupportCursorLoader?

Comment (jdkoren) [inline squidb-addons/squidb-support-loader/src/com/yahoo/squidb/support/SquidCursorLoader.java]: I might be in favor of naming it SquidSupportCursorLoader. Granted the Android way is the support library classes have the same name but different package qualifier as the normal ones, but that lends itself to mistakes IMO.

Comment (sbosley) [inline squidb-addons/squidb-support-loader/src/com/yahoo/squidb/support/SquidCursorLoader.java]: Agreed, I'll update it.

**Comment by jdkoren** [comment]
LGTM

**Pull request merged**